src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Color[] colors = Color.values();
```

Comment:
The 'colors' array is static as well, which is not recommended. Consider making it an instance variable if necessary.